src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -82,7 +82,7 @@ public final class BigQueryUtil {
 
   public static final String BUCKET_PATTERN = "[a-z0-9._-]+";
   public static final String DATASET_PATTERN = "[A-Za-z0-9_]+";
-  public static final String TABLE_PATTERN = "[A-Za-z0-9_-]+";
+  public static final String TABLE_PATTERN = "[\\p{L}\\p{M}\\p{N}\\p{Pc}\\p{Pd}\\p{Zs}]+";
```

Comment:
can you link to documentation that indicates that table name can also include characters in other languages? Also, if this is supported can we add unit tests for table names in other languages?

Reply:
Added a comment
> // Read more about table naming in BigQuery here: https://cloud.google.com/bigquery/docs/tables#table_naming

Added new test cases for valid and invalid table names, including international characters.